dev-python/pycryptodome/pycryptodome-3.4.3.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+# needed to fix the: No such file or directory: 'build/test1.c'
+DISTUTILS_IN_SOURCE_BUILD="1"
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="Self-contained Python package of low-level cryptographic primitives"
+HOMEPAGE="https://github.com/Legrandin/pycryptodome"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+RESTRICT="test"
+
+LICENSE="BSD-2 Unlicense"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="!dev-python/pycrypto"
```

Comment:
This patch will make streamlink use Cryptodome instead of Crypto:

```diff
--- streamlink-0.2.0/setup.py
+++ streamlink-0.2.0/setup.py
@@ -38,7 +38,7 @@
     deps.append("requests>=1.0,!=2.12.0,!=2.12.1,<3.0")
 
 # this version of pycryptodome is known to work and has a Windows wheel for py2.7, py3.3-3.5
-deps.append("pycryptodome==3.4.3")
+deps.append("pycryptodomex==3.4.3")
 
 # When we build an egg for the Win32 bootstrap we don't want dependency
 # information built into it.
--- streamlink-0.2.0/src/streamlink/plugins/beattv.py
+++ streamlink-0.2.0/src/streamlink/plugins/beattv.py
@@ -3,7 +3,7 @@
 from collections import namedtuple
 from io import BytesIO
 try:
-    from Crypto.Cipher import AES
+    from Cryptodome.Cipher import AES
     CAN_DECRYPT = True
 except ImportError:
     CAN_DECRYPT = False
--- streamlink-0.2.0/src/streamlink/stream/hls.py
+++ streamlink-0.2.0/src/streamlink/stream/hls.py
@@ -1,7 +1,7 @@
 from collections import defaultdict, namedtuple
 
 try:
-    from Crypto.Cipher import AES
+    from Cryptodome.Cipher import AES
     import struct
 
     def num_to_iv(n):```

It will allow Crypto and Cryptodome to co-exist, since Cryptodome will be installed under Cryptodomex. However, I think the correct approach would be a virtual/python-crypto package to handle the migration, since pycrypto is abandoned and cryptodome is a drop-in replacement.